todo-app/src/stores/TodoStore.js

```diff
@@ -0,0 +1,108 @@
+import {observable, action, useStrict, runInAction} from 'mobx'
+import Moment from 'moment'
+
+
+useStrict(true)
+
+export default class TodoStore{
+
+    constructor(){
+      this.loadTodos();   // without constructor it won't work, you need to use componentDidMount
+    }
+
+    @observable
+    todos = [];
+
+    @action
+    loadTodos(){
+
+      fetch("https://hyf-react-api.herokuapp.com/todos")
+        .then(response => response.json())
+        .then(response => {
+          // this.setTodos(response)    // if you wanna make the @action tiny [using the function below]
+          console.log("LOAD TODOS : ", response);
+          runInAction(() => {
+            this.todos = response
+          })
+        })
+        .catch(error => {
+          runInAction(() => {
+            console.log(error)
+          })
+        })
+    }
+
+    // @action
+    // setTodos(todos) {
+    //   this.todos = todos
+    // }
+
+    @action  // add a todo item
+    addTodo(text){
+
+      const todo = {
+        "description": text,
+        "deadline": Moment().format('YYYY-MM-DD'),
```

Comment:
First of all, these keys do not need quotes. This is JavaScript, not JSON.

Second, and more importantly, your application has _no way_ to actually set the deadline. This means your application can only create todos that are due today! Feel free to ask some of the other guys for examples. You will need to build this functionality in.

Reply:
Actually, I haven't used the Moment before.
I just wanted to practice/ apply.
Now, I have changed the deadline as always 4 days later.
Moment is great!!